docs/13-configuration.md

```diff
@@ -1,113 +1,128 @@
-````
-
-Note: _This tool automatically registers several JARS for default cloud storage in the spark-default.conf file. If you want to modify this file, simply append any additional JARS to the end of this list_.
+# Configuration Files
+This section refers to the files in the directory *.aztk* that are generated from the `aztk spark init` command.
+
+## *cluster.yaml*
+
+The core settings for a cluster are configured in the *cluster.yaml* file. Once you have set your desired values in *.aztk/cluster.yaml*, you can create a cluster using `aztk spark cluster create`.
+
+This is the default cluster configuration:
+
+```yaml
+# id: <id of the cluster to be created>
+id: spark_cluster
+
+# vm_size: <vm-size, see available options here: https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/>
+vm_size: standard_a2
+
+# size: <number of dedicated nodes in the cluster, not that clusters must contain all dedicated or all low priority nodes>
+size: 2
+
+# size_low_pri: <number of low priority nodes in the cluster, mutually exclusive with size setting>
+
+# username: <username for the linux user to be created> (optional)
+username: spark
+
+# docker_repo: <name of docker image repo (for more information, see https://github.com/Azure/aztk/blob/master/docs/12-docker-image.md)>
+docker_repo: aztk/base:spark2.2.0
+
+# custom_script: <path to custom script to run on each node> (optional)
+
+# wait: <true/false>
+wait: true
+```
+
+Running `aztk spark cluster create` will create a cluster of 4 **Standard\_A2** nodes called 'spark\_cluster' with a linux user named 'spark'. This is equivalent to running the command
+
+```sh
+aztk spark cluster create --id spark --vm-size standard_a2 --size 4 --username spark --wait
+```
+
+NOTE: This assumes that your SSH-key is configured in the *.aztk/secrets.yaml* file.
+
+## *ssh.yaml*
+
+This is the default ssh cluster configuration:
+```yaml
+# username: <name of the user account to ssh into>
+username: spark
+
+# job_ui_port: <local port where the job ui is forwarded to>
+job_ui_port: 4040
+
+# job_history_ui_port: <local port where the job history ui is forwarded to>
+job_history_ui_port: 18080
+
+# web_ui_port: <local port where the spark master web ui is forwarded to>
+web_ui_port: 8080
+
+# jupyter_port: <local port which where jupyter is forwarded to>
+jupyter_port: 8888
+
+# name_node_ui_port: <local port which where Name Node UI is forwarded to>
+name_node_ui_port: 50070
+
+# rstudio_server_port: <local port which where rstudio server is forwarded to>
+rstudio_server_port: 8787
+
+# connect: <true/false, connect to spark master or print connection string (--no-connect)>
+connect: true
+```
+
+Running the command `aztk spark cluster ssh --id <cluster_id>` will ssh into the master node of the Spark cluster. It will also forward the Spark Job UI to localhost:4040, the Spark master's web UI to localhost:8080, and Jupyter to localhost:8888.

```

Comment:
This is also out of date now.

Reply:
yeah --most of this file is out of date. All of the plugin docs update will come with #461 and #467